computeResource.go

```diff
@@ -196,6 +210,24 @@ const (
 // ComputerResourceCreateRequest represents available properties for creating a
 // new compute resource.
 type ComputerResourceCreateRequest struct {
+	Name  string `json:"name,omitempty"`
+	Host  string `json:"host,omitempty"`
+	Login string `json:"login,omitempty"`
+	// SSH port number
+	Port     int                     `json:"port,omitempty"`
+	Type     ComputeResourceAuthType `json:"type,omitempty"`
+	Password string                  `json:"password,omitempty"`
+	// SSH private key
+	Key         string      `json:"key,omitempty"`
+	AgentPort   int         `json:"agent_port,omitempty"`
+	IPBlocks    []int       `json:"ip_blocks,omitempty"`
+	Locations   []int       `json:"locations,omitempty"`
+	LicenseType LicenseType `json:"license_type,omitempty"`
```

Comment:
Can we create CR without license type?

Reply:
yep, if using solus io license model, we can omit "license_type"